pelutils/logging/__init__.py

```diff
@@ -148,6 +151,27 @@ def input(self, prompt: str | Iterable[str] = "") -> str | Generator[str]:
         else:
             return (self._input(p) for p in prompt)
 
+    @classmethod
+    def bool_input(cls, answer: str, *, default: bool) -> bool | None:
```

Comment:
Skulle default have en default-værid?

Reply:
Det synes jeg ikke. Der er ingen særlig præference her for, hvad det skulle være, og jeg tænker, at man nemt kunne indføre fejl, hvis man ikke skal give det eksplicit.